### DISCUSSION TITLE
Account flagged after creating a new Organisation

### DISCUSSION THREAD

**Discussion by taine-zuse** [community/community#53579] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Bug

### Body

Hi all

Iv put a development team together for my project, so I went ahead and paid for Github teams, created a organisation and then transferred all repos to the organisation from my personal repos. Now the issue comes in where it wont allow me to add team members whatsoever stating account flagged, There is also a banner saying there is a problem with my billing however I have received a invoice confirming receipt of payment, the money went off my card and also when I go to billing it even states payment received under payment history and next payment date being a month from the first payment so It does not even give me a option to pay more.

Yet I still cant add team members, iv opened a support ticket with no luck in 5 days and with ALL the repos being added to the organisation there is no way we can actually continue working on them which makes this entire situation extremely frustrating .

Why does Github support take so long to respond to support tickets coming from a paid team account? I thought there was meant to be faster support times etc included in the subscription. I have already uploaded all proof to the ticket in hopes of resolving this quicker but to no avail. Is there anything else I can do ?

**Comment by RamtinKosari** [comment]
Hi Dear @ElegantCodeDev 

I can suggest a few steps you can take to try and expedite the resolution of your problem:

Double-check your payment information: Ensure that the payment method you provided is valid and up-to-date. Sometimes, issues can arise if there are problems with the payment method on file.

Reach out to GitHub support again: If it has been five days without a response, consider opening another support ticket or following up on your existing one. Politely explain the urgency of your situation and provide any additional relevant information or screenshots that may help them understand and address the issue more quickly.

Seek community support: GitHub has a large user community, including forums and discussion boards, where you might find other users who have encountered similar issues. Posting your problem there could potentially attract the attention of someone who has experienced and resolved the same issue.

Explore alternative support channels: Check if GitHub provides any alternative support channels like live chat or phone support. These options may be faster than traditional support tickets.

Consider escalating the issue: If all else fails and you are still not receiving a satisfactory response, you may want to consider reaching out to GitHub's customer support management or exploring any escalation processes they have in place.

Hope that you fix your problem
